analysis/pseudobulk-bulk-prediction/run-prediction.sh

```diff
@@ -57,5 +57,17 @@ for project_dir in $scpca_dir/*; do
       --output_frac_expressed_file "${fraction_expressed_file}"
 done
 
-# Build and export models to results/models/
-Rscript -e "rmarkdown::render('${model-notebooks}/build-assess-models.Rmd')"
+# Build and export models to results/models across different thresholds for expression
+for expr_threshold in -1 0; do
```

Comment:
Just out of curiosity, did you try a small threshold? I'm mostly curious if it works unmodified. If it does not, I would put a note in the notebook where you define the parameter to note that values > 0 don't work (and/or add code to remove the interaction model for those cases).
```suggestion
for expr_threshold in -1 0 0.1; do
```